vulnerabilities/importers/apache_httpd.py

```diff
@@ -81,56 +87,82 @@ def to_advisory(self, data):
                 for version_data in products["version"]["version_data"]:
                     versions_data.append(version_data)
 
-        fixed_version_ranges, affected_version_ranges = self.to_version_ranges(versions_data)
+        fixed_versions = []
+        for timeline_object in data["timeline"]:
+            timeline_value = timeline_object["value"]
+            if "release" in timeline_value:
+                split_timeline_value = timeline_value.split(" ")
+                if "never" in timeline_value:
+                    continue
+                if "release" in split_timeline_value[-1]:
+                    fixed_versions.append(split_timeline_value[0])
+                if "release" in split_timeline_value[0]:
+                    fixed_versions.append(split_timeline_value[-1])
 
-        affected_packages = []
-        fixed_packages = []
-
-        for version_range in fixed_version_ranges:
-            fixed_packages.extend(
-                [
-                    PackageURL(type="apache", name="httpd", version=version)
-                    for version in self.version_api.get("apache/httpd").valid_versions
-                    if SemverVersion(version) in version_range
-                ]
-            )
+        affected_version_range = self.to_version_ranges(versions_data, fixed_versions)
```

Comment:
I would move this line below, just before "if affected_version_range:"

Reply:
Done.